timeline.tex

```diff
@@ -1,17 +1,22 @@
-% Leanne - look to link this to P6 (maybe) 
-\section{Timeline} \label{sec:timeline}
+\section{Roadmap and Timeline} \label{sec:timeline}
 
 This timeline provides a list of key dates  related to the \es program. 
+It will be updated with more dates related to the development of the \esp as they are defined. 
 
-{\bf 2021-11-01:} Issue first version of the \ro Plans for \es document. 
+\textbf{2021-11-12:} Issue first version of the \ro Plans for an \esp. 
 
-{\bf 2023-01-01:} Commissioning Camera (ComCam) on sky
+\textbf{2023-01-01:} Commissioning Camera (ComCam) on sky.
```

Comment:
January 20, 2023 is Engineering First light so the time when comcam is ready to do real stuff on sky.